Automattic/wp-calypso
# Editor: Manage post title edits using global Redux state

This pull request seeks to implement the logic necessary to start assigning edited post values through Redux state, including displaying the current value, editing the value, saving edited values, and considering edited values in Redux state for "dirty" and "empty content" checking. As of this pull request, only title edits will be managed through global state.

The implementation includes compatibility with the Flux approach, specifically in assigning edited values from Redux state to the Flux store prior to saving. This will be removed in future iterations as the editor becomes completely dependent upon Redux state, especially once we start saving the post through Redux action creators.

**Testing instructions:**

Verify that titles can be viewed, updated, and saved in the [post editor](http://calypso.localhost:3000/post). Verify also that the Save option is shown iff the post has any revisions which differ from the saved post, and only if the post has "content" (non-whitespace-only title, content, or excerpt assigned). Confirm expected behavior for new posts and pages, and existing posts and pages.
